toolkits/math/tests/test_arithmetic.py

```diff
@@ -2,49 +2,366 @@
 from arcade.sdk.errors import ToolExecutionError
 
 from arcade_math.tools.arithmetic import (
+    abs_val,
     add,
+    avg,
+    ceil,
+    deg_to_rad,
     divide,
+    factorial,
+    floor,
+    gcd,
+    lcm,
+    log,
+    median,
+    mod,
     multiply,
+    power,
+    rad_to_deg,
+    round_num,
     sqrt,
     subtract,
     sum_list,
     sum_range,
 )
 
 
-def test_add():
-    assert add(1, 2) == 3
-    assert add(-1, 1) == 0
-    assert add(0.5, 10.9) == 11.4
+@pytest.mark.parametrize(
+    "a, b, expected",
+    [
+        ("1", "2", "3"),
+        ("-1", "1", "0"),
+        ("0.5", "10.9", "11.4"),
+        # Big ints
+        ("12345678901234567890", "9876543210987654321", "22222222112222222211"),
+        # Big floats
+        (
+            "12345678901234567890.120",
+            "9876543210987654321.987",
+            "22222222112222222212.107",
+        ),
+    ],
+)
+def test_add(a, b, expected):
+    assert add(a, b) == expected
+
+
+@pytest.mark.parametrize(
+    "a, b, expected",
+    [
+        ("1", "2", "-1"),
+        ("-1", "1", "-2"),
+        ("0.5", "10.9", "-10.4"),
+        # Big ints
+        ("12345678901234567890", "12323456679012345668", "22222222222222222"),
+        # Big floats
+        (
+            "12345678901234567890.120",
+            "12343557689113355768.9079",
+            "2121212121212121.2121",
+        ),
+    ],
+)
+def test_subtract(a, b, expected):
+    assert subtract(a, b) == expected
 
 
-def test_subtract():
-    assert subtract(2, 1) == 1
-    assert subtract(2, 3.5) == -1.5
+@pytest.mark.parametrize(
+    "a, b, expected",
+    [
+        ("-1", "2", "-2"),
+        # TODO(mateo): ok to return negative 0?
+        ("-10", "0", "-0"),
+        ("0.5", "10.9", "5.45"),
+        # TODO(mateo): is scientific notation is acceptable as output?
```

Comment:
probably OK. I believe JS can parse these natively?

Reply:
Yes, JS will handle -0 and float variants gracefully